spec/features/hq/orphan_spec.rb

```diff
@@ -0,0 +1,112 @@
+require 'rails_helper'
+
+RSpec.feature 'Orphan CRUD spec:', :type => :feature do
+
+  background do
+    i_sign_in_as_admin
+    an_orphan_exists 1, "Orphan 1", "Father 1", "2012-01-01", "Male", "Normal"
+    an_orphan_exists 2, "Orphan 2", "Father 2", "2011-01-01", "Female", "High"
+  end
+  
+  scenario 'There should be a list of orphans on the admin index page' do
+    visit hq_orphans_path
+    and_i_should_see "Orphan 1 Father 1"
```

Comment:
What about checking for the name only first and then checking the other details? And having the father in a different line.
```
and_i_should_see "Orphan 1"
and_i_should_see_within "Orphan 1", "Father 1"
and_i_should_see_within "Orphan 1", "2012-01-01"
.....
and_i_should_see "Orphan 2"
....
```